apps/kibana/templates/services/nginx/configs/kibana.conf.j2

```diff
@@ -0,0 +1,49 @@
+{% set http_basic_auth_enabled = activate_http_basic_auth or kibana_activate_http_basic_auth %}
+{% set bypass_htaccess = kibana_nginx_bypass_htaccess_ip_whitelist | length > 0 %}
+upstream kibana-backend {
+  server kibana-app-{{ deployment_stamp }}:{{ kibana_app_port }} fail_timeout=0;
+}
+
+server {
+  listen {{ kibana_nginx_port }};
+  server_name localhost;
+
+  client_max_body_size 100M;
```

Comment:
Can you please add comments about this `100M` choice ? This is huge !

And I don't think upstream load balancer(s) will allow this without configuration change.

Reply:
This was a copy/pasta fail. I've removed it.